docs/setuptools.txt

```diff
@@ -572,6 +572,8 @@ Services and Plugins`_.
 "Eggsecutable" Scripts
 ----------------------
 
+.. warning:: Eggsecutable Scripts are deprecated.
```

Comment:
```suggestion
.. deprecated:: 41.3.0
```

Not actually sure if the next release is going to be a minor release or a major release, so I'm just guessing minor release.